### PR TITLE
Fix issue with kubeadm 1.13

### PR DESCRIPTION
**Pull request by rpothier**
With kubernetes 1.13, the Kubeadm template needs to be updated.

Fixes Issue: #223

### CODE REVIEW

File: image/kubeadm.conf.tmpl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #   SVC_SUBNET, BIND_ADDR
 apiVersion: {{API_VERSION}}
 unifiedControlPlaneImage: mirantis/hypokube:final
-kind: MasterConfiguration
+kind: "{{KUBEADM_KIND}}"
 kubernetesVersion: "{{KUBEADM_VERSION}}"
 api:
```
Comment (ivan4th) [inline image/kubeadm.conf.tmpl]: `api` is now called `apiEndpoint` if I'm not mistaken. Note though that it's still `api` in earlier versions.

Comment (rpothier) [reply image/kubeadm.conf.tmpl]: ok, thanks, Updating now.

**Comment by rpothier** [comment]
@ivan4th I pushed up the change you suggested.

**Comment by ivan4th** [comment]
Thanks!
/approve
/lgtm

**Pull request merged**